packages/markdown-slate/src/slateToCiceroMarkDom.js

```diff
@@ -341,6 +350,34 @@ function handleConditional(node, isTrue, whenTrue, whenFalse) {
     return handleMarks(node,result);
 }
 
+/**
+ * Handles a optional node
+ * @param {*} node the slate variable node
+ * @param {*} hasSome is this optional is present
+ * @param {*} whenSome the nodes when present
+ * @param {*} whenNone the nodes when absent
+ * @returns {*} the ast node
+ */
+function handleOptional(node, hasSome, whenSome, whenNone) {
+    const data = node.data;
+
+    let result = {
+        $class : `${NS_CICERO}.Optional`,
+        name : data.name,
+        nodes: [],
```

Comment:
Does this `nodes` array need to get populated with the text nodes?

Reply:
@DianaLease Good question. This happens naturally as part of the main recursion though. So here:
https://github.com/accordproject/markdown-transform/blob/d88b8900fe1e61e3418bf35c64346adc29f75356/packages/markdown-slate/src/slateToCiceroMarkDom.js#L209